docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,520 @@
+
+        ![Running the application](media/app_run_w.png)
+
+        Alternatively, you can also select the project in the **Project Explorer** view and do one of the following:
+
+        -   Press the **Ctrl + F11** key.
+        -   Click the run icon in the toolbar.
+
+        If you have created multiple emulator instances, select the instance you want from the combo box in the toolbar before selecting to run the application. If you select an offline emulator, it is automatically launched when you select to run the application.
+
+        ![Selecting the emulator to use](media/app_run_multiple_emulators_m.png)
+
+    2.  In the error dialog that appears, to launch the application on the emulator, click **OK**.
+
+        ![Unable to run the addon](media/unable_to_run_addon.png)
+
+        The **WebApp Addon Setting** icon appears in the emulator.
+
+        ![Addon settings app](media/addon_settings_icon.png)
+
+        You can see that the addon is installed named as `tmplDefault` when you run the application.
+
+        ![Addon settings main](media/addon_settings_main.png)
+
+    3.  Now, you can run BasicUI Web application by following the [instruction](../mobile/first-app.md#emulator)
+        
+        > [!NOTE]
+        > You will find nothing different from running the Web application without the addon because there's no visible change in UI. You will make a change on UI later.
+
+For more information on using the emulator features, see [Using Emulator Control Keys, Menu, and Panel](../../../tizen-studio/common-tools/emulator-control-panel.md) and [Using Extended Emulator Features](../../../tizen-studio/common-tools/emulator-features.md).
+
+<a name="target"></a>
+### Running on target device
+
+To run the application on a target device, you can follow the steps similar for the emulator. For more information, see [running BasicUI Web application in a target device](../mobile/first-app.md#target).
+
+In the target device, a BasicUI will appear as expected.
+
+
+<a name="ui"></a>
+## Designing simple UI
+
+The reason that there is no change even after installing the addon above, is that a callback function registered to addon event has only code to print out a log to console.
+
+Now, you can make some changes which is viewable in a Web application.
+
+You need to be familiar with the [basic knowledge in addons](../../guides/applications/addon.md).
+
+### Showing a pop-up on resume
+
+In this section, the example explains how to get the timing of resuming event in the Web application. On the timinig, a pop-up with `Hello. Resume occurred!!` message appears.
+
+First, the JavaScript file `tmplDefault.js` at `shared/res/` is like below:
+
+![JavaScript file for template](media/addon_template_js_file.png)
+
+By now, you must be familiar with the following for addons. For more information, see [basic knowledge in addons](../../guides/applications/addon.md).
```

Comment:
Not clear. 
Are we trying to say:
Following are the events and its callback for addons:

| Used Event     | Matched Callback    |
| -------------- | ------------------- |
| lcPrelaunch | prelaunch |
| lcResume | resume |
| lcSuspend | suspend |
| lcQuit | quit |

For more information, see [basic events in addons](../../guides/applications/addon.md).

Please verify and correct me if I am wrong.

Reply:
Basically, I followed the suggestion at https://github.com/Samsung/tizen-docs/pull/1025#discussion_r378693871

However, your suggestion looks more clear. I've changed in that way.